content/general/events.textile

```diff
@@ -51,25 +51,25 @@ At present, in addition to support for any custom HTTP endpoint, we have ready-m
+* "Messages":/realtime/messages#messages - If the source @channel.message@ is selected, messages trigger function calls as soon as they are published on a channel.
+* "Presence":/realtime/presence - If the source @channel.presence@ is selected, presence events trigger function calls when clients enter, update their data, or leave channels.
+* "Channel lifecycle":/realtime/channels#channel-lifecycle – If the source @channel.lifecycle@ is selected, you receive notifications for channel lifecycle events, such as when a channel is created (following the first client attaching to this channel) or discarded (when there are no more clients attached to the channel).
+* "Occupancy":/realtime/inband-occupancy#occupancy-events - If the source @channel.occupancy@ is selected, you receive notifications for occupancy events, which relate to the number and type of occupants in the channel.
 
 For @channel.lifecycle@ and @channel.occupancy@ metadata in particular, for scalability reasons it is recommended that your rules use these sources, rather than creating @channel.message@ rules on the corresponding "metachannels":/realtime/metachannels.
 
 h2(#configure). Configuring a webhook
 
 Webhooks are configured from the Integrations tab in your "app dashboard":https://faqs.ably.com/how-do-i-access-my-app-dashboard. The following fields are shared between each webhook:
 
-- URL := The URL of the endpoint where messages will be sent
-- Custom headers := Optionally allows you to provide a set of headers that will be included in all HTTP POST requests. You must use format @name:value@ for each header you add, for example, @X-Custom-Header:foo@
-- "Source":#sources := Choose which of @Message@, @Presence@, or @Channel Lifecycle@ events on channels should activate this Reactor Event Rule. @Channel Lifecycle@ events are only available in "Batch Request":#batching mode
-- "Request Mode":#batching := This will either be in @Single Request@ mode or @Batch Request@ mode. "Single Request":#batching will send each event as separately to the endpoint specified by the Rule. "Batch Request":#batching will roll up multiple events in the same request
+- URL := The URL of the endpoint where messages will be sent.
+- Custom headers := Optionally allows you to provide a set of headers that will be included in all HTTP POST requests. You must use format @name:value@ for each header you add, for example, @X-Custom-Header:foo@.
+- "Source":#sources := Choose which of @channel.message@, @channel.presence@, @channel.lifecycle@, or @channel.occupancy@ events on channels should activate this Event Rule.
+- "Request Mode":#batching := This will either be in @Single Request@ mode or @Batch Request@ mode. "Single Request":#batching will send each event as separately to the endpoint specified by the Rule. "Batch Request":#batching will roll up multiple events in the same request.
```

Comment:
"each event as separately"

Reply:
"Rule" should not be capitalised IMO .. although we probably need a ticket for this